crates/hc_sandbox/src/cmds.rs

```diff
@@ -80,6 +80,9 @@ pub enum NetworkType {
     WebRTC {
         /// URL to a holochain tx5 WebRTC signal server.
         signal_url: String,
+
+        /// Optional override webrtc peer connection config to use.
+        webrtc_config: Option<String>,
```

Comment:
This is accepting JSON on the command line? Would probably prefer to pass a path to a config file

Reply:
Done: https://github.com/holochain/holochain/pull/3842/commits/9e85527e9a3b0130d4a454fdff52fb68f2b5b5a5